### DISCUSSION TITLE
How to move objects using transition

### DISCUSSION THREAD

**Discussion by lucaslimaunb** [Moddable-OpenSource/moddable#1008] [Q&A]
I'm looking for more information. On documentation I found only one example using transition, and it change colors only. I used the method moveby inside of the onstep but the performance is not good. There is another way to do it? Somebody can send some examples of usage?

**Comment by andycarle** [comment]
Hi @lucaslimaunb,

Take a look at the [timeline example](https://github.com/Moddable-OpenSource/moddable/tree/public/examples/piu/timeline). The [transitions in that app](https://github.com/Moddable-OpenSource/moddable/blob/public/examples/piu/timeline/main.js) show how to move objects about on the screen.

If that's not roughly what you had in mind, let me know.

**Comment by lucaslimaunb** [comment]
Hello @andycarle!

I wanna something to move an container on transition to up and another container comes from the bottom of screen. 
[Here](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/piu/piu.md#animations) on the documentation says that transition is the best for simple animations to move between screens. I already made some animations using the timeline or drawing on port content but is possible to make animations moves using the transition itself?

**Comment by andycarle** [comment]
Ah, I see. 

Most of our examples use the `Transition` object indirectly by using one of the Piu built-in transitions like `WipeTransition` or `CombTransition`. To see those in action, try out the [transitions example](https://github.com/Moddable-OpenSource/moddable/tree/public/examples/piu/transitions) or the [bars example](https://github.com/Moddable-OpenSource/moddable/tree/public/examples/piu/bars). 

The transitions example is more comprehensive, but might be a little harder to follow. The bars example is pretty straightforward.
